POC/GPT4V-OCR.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Do you mean use the same images from TrOCR evaluation set? By "transcribe" I assume you mean recognize.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/ml-herbarium/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Yes, I wanted to test GPT4-V on the TrOCR images. Yes, by transcribe I mean recognize the text.</p>